### PR TITLE
#158626114 Add login with social networks media

### PR DESCRIPTION
**Pull request by muhozi**
#### What does this PR do?
- Add social media logins
#### Description of Task to be completed?
For now, users can log in on wger using their social media networks(**Twitter**, **Facebook** and **Google**).
#### How should this be manually tested?
On the login page, click on login with facebook, twitter or google buttons
#### What are the relevant pivotal tracker stories?
[#158626114](https://www.pivotaltracker.com/story/show/158626114)
#### Screenshots (if appropriate)
screen shot 2018-07-19 at 19 42 29
screen shot 2018-07-19 at 15 46 09
screen shot 2018-07-19 at 19 59 53
screen shot 2018-07-19 at 18 49 47

### CODE REVIEW

File: wger/core/views/user.py [hunk 1]
```diff
@@ -540,3 +547,180 @@ def get_context_data(self, **kwargs):
             context['object_list']['members']
         }
         return context
+def oauthLogin(request, platform):
+    '''
+        Social networks login
+    '''
+    authorize_url = '{}/en/user/login/oauth/facebook/authorize'.format(os.getenv('SITE_URL'))
```
Comment (parseen254) [inline wger/core/views/user.py]: why is this outside of a condition?

File: wger/core/views/user.py [hunk 2]
```diff
@@ -540,3 +547,180 @@ def get_context_data(self, **kwargs):
...
+    if platform=='facebook':
+        # Redirect to facebook login page
+        url = 'https://www.facebook.com/v3.0/dialog/oauth?client_id={}&redirect_uri={}&state={}&scope={}'.format(
+                '470369423388367',
```
Comment (parseen254) [inline wger/core/views/user.py]: this is sensitive data and should not be pushed to repo.

Comment (muhozi) [reply wger/core/views/user.py]: @parseendavid Thank you for this. Let me remove it.

File: wger/core/views/user.py (hunk 1 above)
Comment (muhozi) [reply wger/core/views/user.py]: Yeah, It should be inside the condition. I missed that. Thanks.

**Pull request merged**